en/development/testing.rst

```diff
@@ -415,6 +415,9 @@ they are defined. The format used to define these fields is the same used with
         - ``string``: maps to ``VARCHAR``
         - ``text``: maps to ``TEXT``
         - ``biginteger``: maps to ``BIGINT``
+        - ``smallinteger``: maps to ``SMALLINT``
+        - ``tinyinteger``: maps to ``TINYINT`` or ``SMALLINT`` depending on the
+          database platform.
```

Comment:
> boolean: maps to TINYINT

We should explain that better,now.

> Maps to BOOLEAN except in MySQL, where TINYINT(1) is used to represent booleans.

https://book.cakephp.org/2.0/en/models/model-attributes.html#schema